webrtc.html

```diff
@@ -4894,6 +4894,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
              RTCRtcpParameters                         rtcp;
              sequence&lt;RTCRtpCodecParameters&gt;           codecs;
              RTCDegradationPreference                  degradationPreference = "balanced";
+             boolean                                   upscaleAllowed = false;
```

Comment:
If we decide to add a flag for upscaling I think it should be named something like "spatialUpscaleAllowed" or so since the only upscale where any document (in this case JSEP) has text forbidding it as default is for video width and height.